src/plots/barPlot.ts

```diff
@@ -291,6 +291,7 @@ export module Plots {
     }
 
     protected _visibleOnPlot(datum: any, pixelPoint: Point, selection: d3.Selection<void>): boolean {
+      Utils.Window.deprecated("v1.1.0");
```

Comment:
What do you mean with automate? Also, I am aiming to release this in v1.1, i.e. demo it on Friday